test/app/test/interactive.clj

```diff
@@ -22,3 +41,19 @@
                (verify-call-times-for println 2)
                (verify-first-call-args-for println
                                            no-classes-loaded-error)))))
+
+(deftest can-execute-mc-operation
+  (let [javaparser-cus {:cus (take 2 (cus javaparser-cus-path))}
```

Comment:
In this case, it does not matter. We just want to make sure that the function 'printOperation' is called with the appropriate parameters from the interactive mode namespace. However, I think it's important to clarify this. I will add some comments.